docs/content/en/docs/concepts/eksafeatures.md

```diff
@@ -51,3 +51,16 @@ To learn more about Amazon EKS, see [Amazon Elastic Kubernetes Service](https://
 | Control plane pricing                       | Free to download, paid support subscription option  | Hourly pricing per cluster |
 | AWS Support                       | Additional annual subscription (per cluster) for AWS support | Basic support included. Included in paid AWS support plans (developer, business, and enterprise)  |
 ||||
+
+### Comparing Amazon EKS Anywhere to Amazon EKS on Outposts
+
+Like EKS Anywhere, Amazon EKS on Outposts provides a means of running Kubernetes clusters using EKS software and customer on-premises hardware.
+The main difference is that Amazon provides the hardware with Outposts, while most EKS Anywhere providers leverage the customer's own hardware.
```

Comment:
To me, there are two main differences: the one you mentioned, and the second one is - EKS on Outpost provides an AWS-managed Kubernetes control plane versus with EKS-A, customers have to self-manage the Kubernetes control plane

Reply:
+1 to what Lichun says. 

With Amazon EKS on Outposts, the Kubernetes control plane is fully managed by AWS. With EKS Anywhere, customers are responsible for managing the lifecycle of the Kubernetes control plane with EKS Anywhere automation tooling.

Reply:
Done.